### PR TITLE
Introduce coverity scan parser

### PR DESCRIPTION
**Pull request by tschaepe-secuvera**
Add a parser for the Coverity SAST tool.

There is already a parser for Coverity but it covers only the JSON output from an outdated Coverity Connect endpoint (v2 is the current version).

Nevertheless it is possible to get a JSON report directly from the CLI with `coverity scan --project-dir <project_dir> --local <result_file> --local-format json`.

https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/cli/topics/commit_configuration.html#local_configuration

### CODE REVIEW

File: docs/content/en/integrations/parsers/file/coverity_scan.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Coverity Scan JSON Report"
+toc_hide: true
+---
+Run `coverity scan --project-dir <project_dir> --local <result_file> --local-format json` to create the JSON report.
```
Comment (manuel-sommer) [inline docs/content/en/integrations/parsers/file/coverity_scan.md:5]: Could you please also add a link to the Scanner

Comment (mtesauro) [reply docs/content/en/integrations/parsers/file/coverity_scan.md:5]: +1

**Review by mtesauro — APPROVED** [review]
Approved

File: docs/content/en/integrations/parsers/file/coverity_scan.md (hunk 1 above)
Comment (tschaepe-secuvera) [reply docs/content/en/integrations/parsers/file/coverity_scan.md:11]: I added a link to vendor and to the documentation.

**Review by grendel513 — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**